bugsnag-android-core/src/main/java/com/bugsnag/android/BugsnagStoreMigrator.kt

```diff
@@ -0,0 +1,32 @@
+package com.bugsnag.android
+
+import java.io.File
+
+class BugsnagStoreMigrator {
+
+    fun moveToNewDirectory(persistenceDir: File) {
```

Comment:
If the `BugsnagStoreMigrator` is an `object` this method can be marked `@JvmStatic`